tests/acceptance/17_packages/10_new/unsafe/README

```diff
@@ -0,0 +1,10 @@
+Run this generator and redirect the output to the resulting policy file.
```

Comment:
@nickanderson I added a README and added code to the generator to exit if not python 3.